agent/src/model.rs

```diff
@@ -40,6 +68,11 @@ pub struct Region {
     pub key_pem: Option<String>,
 
     pub root_pem: Option<String>,
+    // If this region was created as part of a clone.
+    pub source: Option<SocketAddr>,
+
+    // If this region is read only
+    pub read_only: bool,
```

Comment:
We may be able to get away with using `#[serde(default)]` for these two fields, and not have to split into RegionV1 vs Region. That should deserialize the existing data files without errors.

Reply:
Yeah, I got something to work there.